src/pages/hcp/api-docs/packer/[[...page]].tsx

```diff
@@ -0,0 +1,145 @@
+import type { InferGetStaticPropsType } from 'next'
+
+import SidebarSidecarLayout from 'layouts/sidebar-sidecar'
+import { OpenApiPageContents } from 'components/open-api-page'
+/* Used server-side only */
+import { cachedGetProductData } from 'lib/get-product-data'
+import { isDeployPreview } from 'lib/env-checks'
+import fetchGithubFile from 'lib/fetch-github-file'
+import {
+	getPathsFromSchema,
+	getPropsForPage,
+	processSchemaString,
+	processSchemaFile,
+} from 'components/open-api-page/server'
+import {
+	generateProductLandingSidebarNavData,
+	generateTopLevelSidebarNavData,
+} from 'components/sidebar/helpers'
+
+import type { PageWithLayout } from 'types/layouts'
+
+const productSlug = 'hcp'
+const targetFile = {
+	owner: 'hashicorp',
+	repo: 'cloud-api',
+	path: 'specs/cloud-packer-service/stable/2021-04-30/hcp.swagger.json',
+}
+
+// The path to read from when running local preview in the context of the cloud.hashicorp.com repository
+// - This is unused as cloud.hashicorp.com does not use the dev-portal website shell yet
+const targetLocalFile = '../TODO'
```

Comment:
I'm not sure this page will ever ready from the file system 🤔 We might be able to just remove references to a local file here.

Reply:
My thought was that we maybe eventually want a local fixture to fetch since the current Octokit-fetching with a shared GH PAT is bound to be hit by rate limits

Reply:
I'll go ahead and delete this for now